app/components/Channels/ChannelCardList.js

```diff
@@ -38,19 +37,18 @@ const ChannelCardList = ({
         pr={index % 2 ? 4 : 3}
         pl={index % 2 ? 3 : 4}
         pt={3}
-        mb={ROW_MARGIN_BOTTOM}
+        pb={ROW_PADDING_BOTTOM}
         style={style}
         key={key}
       >
-        <Card>
-          <ChannelCardListItem
-            channel={channel}
-            currencyName={currencyName}
-            openModal={openModal}
-            setSelectedChannel={setSelectedChannel}
-            networkInfo={networkInfo}
-          />
-        </Card>
+        <ChannelCardListItem
+          channel={channel}
+          currencyName={currencyName}
+          openModal={openModal}
+          setSelectedChannel={setSelectedChannel}
+          networkInfo={networkInfo}
+          css={{ height: '100%' }}
```

Comment:
I wonder what's the reason behind this change ? We have a static row height of the list items

Reply:
In the basic card view (shown on the list page), there are 3 rows of data. In some cases, we don't have values for all of these rows (eg, force closing channels).

Previously, we should show the rows but with values `unknown`, which looked a bit strange and raises the question of why the value is unknown.

In this PR, this is changed so that we only show data rows when we actually have values to show. This means that rather than always being 3 rows, we now display *up to* 3 rows.

In the case where there is only 1 row, we want to ensure that the card still takes up all available height.